elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -152,15 +173,18 @@ RUN yum update -y && \
     yum install -y nc unzip wget which && \
     yum clean all
 COPY CENTOS_LICENSING.txt /root
-COPY --from=prep_es_files --chown=1000:0 /opt/jdk-12.0.2 /opt/jdk-12.0.2
+##COPY --from=prep_es_files --chown=1000:0 /opt/jdk-12.0.2 /opt/jdk-12.0.2
 
-ENV JAVA_HOME /opt/jdk-12.0.2
+COPY --from=prep_es_files --chown=1000:0 /opt/jdk /opt/jdk
+ENV JAVA_HOME /opt/jdk
+ENV PATH $PATH:$JAVA_HOME/bin
+RUN java -version
```

Comment:
could we use the jdk located in the es folder?

Reply:
That is exactly the one, bundled with ES.

Reply:
could we avoid this copy?

Reply:
This is no difference compares to copy directly from the ES because it is from the ES originally.